gradle.properties

```diff
@@ -5,7 +5,7 @@ jetBrainsChannel=stable
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 # if plugin uses intellij-common-ui-test library, 'platformType' and 'platformVersion' variables names MUST be used
 # platformType = IC (not needed as hard-coded in gradle build directly)
-platformVersion=2024.2
+platformVersion=2024.3
```

Comment:
please also update (again) the matrix validation workflow to use this version, too